pkgs/servers/monitoring/zabbix/versions.nix

```diff
@@ -1,4 +1,9 @@
 generic: {
+  v64 = generic {
```

Comment:
usually we don't like packaging non LTS versions of `zabbix` because their lifespan is so short and expires before our (NixOS) next release... but in this case it looks like EOL isn't until december.... who is going to remember to clean this version up in december?

what is the motivation to add this and the plan to clean it up?

Reply:
The motivation is an incompatibility of my previous docker zabbix (running 6.4) and the 6.0 version. 
If needed, I will remove it, once its EOL